returnn/tf/layers/basic.py

```diff
@@ -6114,14 +6116,21 @@ def __init__(
         filter=None,
         filter_perm=None,
         bias=None,
+        use_time_mask=None,
         **kwargs,
     ):
         """
         :param Sequence[Dim]|Sequence[int] filter_size: (width,), (height,width) or (depth,height,width)
             for 1D/2D/3D conv.
             The input data ndim must match, or you can add dimensions via input_expand_dims or input_add_feature_dim.
             It will automatically swap the batch-dim to the first axis of the input data.
-        :param str padding: "same" or "valid"
+        :param str padding: "same", "valid" or "same_static".
+            "same_static" is calculated differently depending on whether an axis is static or dynamic.
+            For static axes, "same_static" padding is the same as "same" padding,
+            i.e. filter_size - 1 - (T + strides - 1) % strides.
+            For dynamic axes, "same_static" calculates the total padding size as
+            filter_size - 1, i.e. it is independent of the length T of the axis and the striding. To avoid skipping
+            any frames on the right, we set left_padding = (filter_size - strides) // 2.
```

Comment:
Make proper newlines.
```suggestion
            filter_size - 1, i.e. it is independent of the length T of the axis and the striding.
            For dynamic axes, to avoid skipping any frames on the right,
            we set left_padding = (filter_size - strides) // 2.
```

Reply:
Not done yet?